public/components/FeatureAnywhereContextMenu/AssociatedDetectors/containers/AssociatedDetectors.tsx

```diff
@@ -0,0 +1,335 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useMemo, useEffect, useState } from 'react';
+import {
+  EuiFlyoutHeader,
+  EuiTitle,
+  EuiSpacer,
+  EuiInMemoryTable,
+  EuiFlyoutBody,
+  EuiButton,
+  EuiFlyout,
+  EuiFlexItem,
+  EuiFlexGroup,
+} from '@elastic/eui';
+import { get, isEmpty } from 'lodash';
+import '../styles.scss';
+import { getColumns } from '../utils/helpers';
+import { useDispatch, useSelector } from 'react-redux';
+import { AppState } from '../../../../redux/reducers';
+import { DetectorListItem } from '../../../../models/interfaces';
+import { getSavedFeatureAnywhereLoader } from '../../../../services';
+import {
+  GET_ALL_DETECTORS_QUERY_PARAMS,
+  SINGLE_DETECTOR_NOT_FOUND_MSG,
+} from '../../../../pages/utils/constants';
+import { getDetectorList } from '../../../../redux/reducers/ad';
+import {
+  prettifyErrorMessage,
+  NO_PERMISSIONS_KEY_WORD,
+} from '../../../../../server/utils/helpers';
+import { SavedObjectLoader } from '../../../../../../../src/plugins/saved_objects/public';
+import {
+  EmptyAssociatedDetectorMessage,
+  ConfirmUnlinkDetectorModal,
+} from '../components';
+import { ISavedAugmentVis } from '../../../../../../../src/plugins/vis_augmenter/public';
+import { ASSOCIATED_DETECTOR_ACTION } from '../utils/constants';
+
+interface ConfirmModalState {
+  isOpen: boolean;
+  action: ASSOCIATED_DETECTOR_ACTION;
+  isListLoading: boolean;
+  isRequestingToClose: boolean;
+  affectedDetector: DetectorListItem;
+}
+
+function AssociatedDetectors({ embeddable, closeFlyout, core, setMode }) {
```

Comment:
Please avoid passing core as a prop around as tyler suggested in my previous pr - https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/482#discussion_r1198345354